jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
I've been meaning to switch to http://vcversioner.readthedocs.org/en/latest/ which handles that process for us and isn't overcomplex like some other tools that do the same sort of thing.

Good to hear that we can swap those.